specification/_types/Errors.ts

```diff
@@ -17,56 +17,35 @@
  * under the License.
  */
 
-import { PainlessExecutionPosition } from '@global/scripts_painless_execute/types'
-import { Dictionary } from '@spec_utils/Dictionary'
-import { HttpHeaders, Id, Ids, IndexName, Uuid } from './common'
+import { Id, IndexName } from './common'
 import { integer, long } from './Numeric'
+import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
+import { AdditionalProperties } from '@spec_utils/behaviors'
```

Comment:
Is `AdditionalProperties` used for anything else? The name feels cumbersome in this context, and I'm wondering if we can pick something shorter, and possibly avoid the word "additional".

Reply:
Additional properties is a "behavior", interfaces that capture things that can't be fully expressed in TypeScript.

That one is used to indicate that a class has a set of well-known properties with each their specific types (those described in the class) but also accepts arbitrary properties of a given type. It's heavily used in aggregations.

In this context the type of these additional properties is anything (`UserDefinedValue`), but in other contexts the type is well-defined (see [Aggregate.ts](https://github.com/elastic/elasticsearch-specification/blob/main/specification/_types/aggregations/Aggregate.ts))

Behaviors also trigger specialized code generation schemes, so a renaming is possible (naming is hard) but would have to be coordinated with updates in the code generators.

More details at https://github.com/elastic/elasticsearch-specification/blob/main/docs/behaviors.md